couch/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 <!-- towncrier release notes start -->
 
+## Unreleased
```

Comment:
These days we automatically generate the changelog. Please use the [relevant ddev command](https://datadoghq.dev/integrations-core/ddev/cli/#ddev-release-changelog-new) to generate a snippet.